idealadarsh/app.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask, request, redirect, render_template
+
+app = Flask(__name__)
+
+@app.route("/")
+def index():
+    return render_template('index.html')
+
+@app.route('/result', methods = ['POST'])
+def result():
+    name = request.form['name']
```

Comment:
If I press submit button, without entering any name than it's should prompt the user that his/her name is missing

Reply:
@idealadarsh Send PR fixing this, so I think before sending another PR you should read https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/